app/errors.js

```diff
@@ -5,3 +5,18 @@ const internalError = (message, internalCode) => ({
 
 exports.DEFAULT_ERROR = 'default_error';
 exports.defaultError = message => internalError(message, exports.DEFAULT_ERROR);
+
+exports.DATABASE_ERROR = 'database_error';
+exports.databaseError = message => internalError(message, exports.DATABASE_ERROR);
+
+exports.EMAIL_ERROR = 'email_is_invalid';
```

Comment:
Check all file, i see validations messages and errors messages (only errors messages should be here)

Reply:
Validations messages could be in `validations` file